src/shared/utils/ast-jsx-utils.ts

```diff
@@ -340,3 +340,13 @@ const convertToUnaryOperator = (operation: string): UnaryOperation => {
     return '!'
   }
 }
+
+export const createTernaryOperation = (
```

Comment:
all publicly exported functions from ast utils have the last optional param set like this
```t = types```

this is useful for potentially mocking it when unit testing the functions. please add this last param to keep the consistency. it's the only change needed, as this is not supplied from the outside yet

Reply:
Updated 👍